frontend/src/components/Navbar.tsx

```diff
@@ -80,7 +80,6 @@ function Navbar(props: any) {
     letterCodes: [],
     numberRange: [MIN_COURSE_CODE, MAX_COURSE_CODE],
   };
-  const theme = useTheme();
```

Comment:
rip theme #202020
hope to never see you again

Reply:
one themey boy ruined a whole sprint -.-